config/step/ld_index.yaml

```diff
@@ -2,5 +2,6 @@ _target_: otg.ld_index.LDIndexStep
 
 session:
   start_hail: true
-
+grch37_to_grch38_chain_path: ${datasets.chain_hail_37_38}
+ld_index_raw_template: ${datasets.ld_index_raw_template}
```

Comment:
This change doesn't look right in relation to the changes in `config/datasets/gcp.yaml`: while we are removing `chain_hail_38_37` from `gcp.yaml`, we are adding `datasets.chain_hail_37_38`. This looks like it wouldn't work because the line in `gcp.yaml` is removed

Reply:
Great catch! Thanks. These params are not used after https://github.com/opentargets/genetics_etl_python/pull/233/files